### ISSUE TITLE
rebuild called twice

### ISSUE THREAD

**Issue by ivosabev** [mapbox/node-pre-gyp#444]
This is really curious case, the rebuild command is called twice in my project packages folder, but if I create a new folder outside my repo and run the install works fine. I am installing `node_or_tools`. In both cases the package.json is clean and there are no other dependencies or scripts or any special clauses inside.

package.json
```
{
  "name": "xxx",
  "version": "0.0.1",
  "description": "",
  "main": "index.js",
  "author": "",
  "license": "MIT",
  "dependencies": {
    "node_or_tools": "1.0.6"
  }
}
```

Working in a regular folder
```
node-pre-gyp verb command install []
node-pre-gyp WARN Using needle for node-pre-gyp https download 
node-pre-gyp info check checked for "/xxx/not-project/node_modules/node_or_tools/lib/binding/node_or_tools.node" (not found)
node-pre-gyp http GET https://mapbox-node-binary.s3.amazonaws.com/node_or_tools/v1.0.6/Release/node-v64-darwin-x64.tar.gz
node-pre-gyp http 403 https://mapbox-node-binary.s3.amazonaws.com/node_or_tools/v1.0.6/Release/node-v64-darwin-x64.tar.gz
node-pre-gyp WARN Tried to download(403): https://mapbox-node-binary.s3.amazonaws.com/node_or_tools/v1.0.6/Release/node-v64-darwin-x64.tar.gz 
node-pre-gyp WARN Pre-built binaries not found for node_or_tools@1.0.6 and node@10.14.2 (node-v64 ABI, unknown) (falling back to source compile with node-gyp) 
node-pre-gyp http 403 status code downloading tarball https://mapbox-node-binary.s3.amazonaws.com/node_or_tools/v1.0.6/Release/node-v64-darwin-x64.tar.gz 
node-pre-gyp verb command build [ 'rebuild' ]
```

Not working in a lerna project monorepo packages folder
```
node-pre-gyp verb command install []
node-pre-gyp WARN Using request for node-pre-gyp https download 
node-pre-gyp info check checked for "/xxx/project/packages/server/node_modules/node_or_tools/lib/binding/node_or_tools.node" (not found)
node-pre-gyp http GET https://mapbox-node-binary.s3.amazonaws.com/node_or_tools/v1.0.6/Release/node-v64-darwin-x64.tar.gz
node-pre-gyp http 403 https://mapbox-node-binary.s3.amazonaws.com/node_or_tools/v1.0.6/Release/node-v64-darwin-x64.tar.gz
node-pre-gyp WARN Tried to download(403): https://mapbox-node-binary.s3.amazonaws.com/node_or_tools/v1.0.6/Release/node-v64-darwin-x64.tar.gz 
node-pre-gyp WARN Pre-built binaries not found for node_or_tools@1.0.6 and node@10.14.2 (node-v64 ABI, unknown) (falling back to source compile with node-gyp) 
node-pre-gyp http 403 status code downloading tarball https://mapbox-node-binary.s3.amazonaws.com/node_or_tools/v1.0.6/Release/node-v64-darwin-x64.tar.gz 
node-pre-gyp verb command build [ 'rebuild' ]
node-pre-gyp WARN Pre-built binaries not installable for node_or_tools@1.0.6 and node@10.14.2 (node-v64 ABI, unknown) (falling back to source compile with node-gyp) 
node-pre-gyp WARN Hit error Connection closed while downloading tarball file 
node-pre-gyp verb command build [ 'rebuild' ]
```

**Comment by nicolasnoble** [comment]
Which version of node-pre-gyp are you using? I fixed a double-rebuild bug in 0.12.

**Comment by ivosabev** [comment]
It is (https://github.com/mapbox/node-or-tools/blob/master/package.json#L25)[0.10]. I've submitted a PR there to bump to `0.12`. Thanks for pointing that out and fixing it, it has been bothering us for months.
